### PR TITLE
list homework

### PR DESCRIPTION
**Pull request by gummer11**
Add
-ordered list
-unordered list
-description list
-ordered list with a nested unordered list

### CODE REVIEW

File: lists/Gummer_lists/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <dd>Dog, Chicken,Parrot</dd>
+
+
+</dl>
+
+<hr>
+    <!-- ordered list with a nested unordered list -->
+    <ol>
+        <li>Austria</li>
+        <ul>
```
Comment (virag-ky) [inline lists/Gummer_lists/index.html:41]: Almost right, you just have to wrap the `ul` tags inside an `li` tag, remember, ordered and unordered lists can only contain list items (li), but then the `li` tags can contain additional elements.

**Comment by gummer11** [comment]
Thank you Virag the notification, I have changed, I hope it is better now! :)

**Comment by virag-ky** [comment]
Yes, it's perfect now, thank you for your changes. 👍🏻

**Comment by gummer11** [comment]
I thank you for you notice! 😊

**Pull request merged**